packages/forms/src/styled/text/StyledTextFauxInput.ts

```diff
@@ -18,6 +24,26 @@ export interface IStyledTextFauxInputProps extends IStyledTextInputProps {
   isReadOnly?: boolean;
 }
 
+export function getValidationHue(
```

Comment:
This local utility can be used across forms components. As of now it is a style helper for now.